pkg/simulator/terraform.go

```diff
@@ -78,6 +78,9 @@ func (s *Simulator) InitIfNeeded() error {
 		}
 		accessCIDR = *ip + "/32"
 	}
+
+	var accessUsername string = ""
```

Comment:
If this is always empty why do we have it and propagate it throughout the binary?

Might be worth using this to read from a file instead? That way we can have a file in `.kubesim` people can add usernames to instead of using the cli?

Reply:
true, might be good to raise a PR for this one as it applies to the accessCIDR too and requires some structural changes then